libc/src/math/generic/CMakeLists.txt

```diff
@@ -1816,6 +1816,19 @@ add_entrypoint_object(
     -O2
 )
 
+add_entrypoint_object(
+  nextafterf128
+  SRCS
+    nextafterf128.cpp
+  HDRS
+    ../nextafterf128.h
+  DEPENDS
+    libc.src.__support.macros.properties.types
+    libc.src.__support.FPUtil.manipulation_functions
+  COMPILE_OPTIONS
+    -O2
```

Comment:
You can make all the `nextafter` functions `-O3`.  We were a bit conservative with the optimization level at the beginning.